app/commands/exercise/representation/send_new_feedback_notifications.rb

```diff
@@ -0,0 +1,36 @@
+class Exercise::Representation::SendNewFeedbackNotifications
+  include Mandate
+
+  queue_as :notifications
+
+  initialize_with :representation
+
+  def call = iterations.each { |iteration| send_notification(iteration) }
+
+  private
+  def iterations
+    if representation.has_essential_feedback?
+      latest_active_iterations
+    elsif representation.has_actionable_feedback?
+      latest_recent_active_iterations
+    else
+      raise "Incorrect feedback type"
+    end
+  end
+
+  def submission_representations = representation.submission_representations
```

Comment:
This should `delegate`. I would have expected rubocop to sort that for you automatically. Maybe it can't for one line methods? Or maybe you still have it not running on commit?

Reply:
No, it shouldn't have. This is in the `private` section, which is when Rubucop doesn't suggest `delegate` as I think that would make it public.